fix(push,cli): fixed push & gitflow command

leorolland opened the pull request:
- Removed alias
- added /usr/bin/gitflow script
- fixed push command
- reimplemented gitlab.py instanciation strategy

ryshu commented on gitflow_toolbox/common/gitlab.py:
```diff
@@ -41,9 +37,11 @@ def project(self) -> Project:
 
     def __init__(self):
-        super().__init__(get_env("REMOTE_GITLAB_URL"), get_env("REMOTE_GITLAB_PRIVATE_TOKEN"))
+        self.private_token = get_env("REMOTE_GITLAB_PRIVATE_TOKEN", "GITLAB_PRIVATE_TOKEN", "CI_JOB_TOKEN")
```
```suggestion
        self.__private_token = get_env("REMOTE_GITLAB_PRIVATE_TOKEN", "GITLAB_PRIVATE_TOKEN", "CI_JOB_TOKEN")
```

ryshu commented on gitflow_toolbox/common/gitlab.py:
```diff
@@ -10,15 +10,11 @@
-            super().__init__(get_env("CI_SERVER_URL"), private_token)
-        else:
-            super().__init__(get_env("GITLAB_URL"), get_env("GITLAB_PRIVATE_TOKEN"))
+        self.private_token = get_env("GITLAB_PRIVATE_TOKEN", "CI_JOB_TOKEN")
```
```suggestion
        self.__private_token = get_env("GITLAB_PRIVATE_TOKEN", "CI_JOB_TOKEN")
```

leorolland replied:
`private_token` attribute is used to generate a pre-authenticated url for `push` (push.py) and so needs to be accessible

leorolland replied:
Maybe we can refactor the authenticated URL generation into a class method :)

ryshu replied:
No, it's ok for me

leorolland committed:
fix(push,cli): fixed push & gitflow command

Removed alias, added /usr/bin/gitflow script, fixed push command, reimplemented gitlab.py instanciation strategy.
